packages/vite/src/node/plugins/asset.ts

```diff
@@ -204,6 +204,10 @@ export function assetPlugin(config: ResolvedConfig): Plugin {
           config.command === 'build' && this.getModuleInfo(id)?.isEntry
             ? 'no-treeshake'
             : false,
+        meta:
+          config.command === 'build' && this.getModuleInfo(id)?.isEntry
+            ? { 'vite:asset': true }
```

Comment:
Should we name this `'vite:entry-asset'`? Or maybe remove the `isEntry` condition and later on let other places use the `'vite:asset'` meta info?
It may be confusing while debugging for people to see `vite:asset` only added for some assets handled by vite.

Reply:
As a convention, [the plugin name is used as a key name](https://rollupjs.org/plugin-development/#custom-module-meta-data:~:text=Note%20the%20convention%20that%20plugins%20that%20add%20or%20modify%20data%20should%20use%20a%20property%20corresponding%20to%20the%20plugin%20name%2C%20in%20this%20case%20annotating.).
I'll remove the `isEntry` condition.